lib/Dialect/FIRRTL/Transforms/CreateSiFiveMetadata.cpp

```diff
@@ -32,6 +32,19 @@ static const char metadataDirectoryAnnoClass[] =
     "sifive.enterprise.firrtl.MetadataDirAnnotation";
 
 namespace {
+
+// TODO: Move this to firrtl/utility
+StringAttr getOrAddInnerSym(Operation *op) {
+  auto attr = op->getAttrOfType<StringAttr>("inner_sym");
+  if (attr)
+    return attr;
+  ModuleNamespace modNamespace(op->getParentOfType<FModuleOp>());
+  auto name = modNamespace.newName("s_");
```

Comment:
we don't have an ascii shortage, a couple more letters to indicate where this symbol came from would be good.